fixtures/datagen/fakerize.py

```diff
@@ -0,0 +1,446 @@
+    "Project",
+    "Gambling",
+    "Miner",
+    "Mixer",
+    "Individual",
+    "Other",
+]
+FAKE_VASPS = {
+    "CharlieBank": "SUBMITTED",
+    "Delta Assets": "APPEALED",
+    "Echo Funds": "SUBMITTED",
+    "Foxtrot LLC": "VERIFIED",
+    "GolfBucks": "ERRORED",
+    "Hotel Corp": "VERIFIED",
+    "IndiaCoin": "VERIFIED",
+    "Juliet Capulet LLC": "PENDING_REVIEW",
+    "KiloVASP": "VERIFIED",
+    "Lima Beancounters": "REJECTED",
+    "Mikes Official VASP": "REJECTED",
+    "NovemberCash": "VERIFIED",
+    "Romeo Montague Labs LLC": "VERIFIED",
+    "Oscar Inc": "PENDING_REVIEW",
+}
+STATE_CHANGES = {
+    "SUBMITTED": {
+        "previous_state": "NO_VERIFICATION",
+        "current_state": "SUBMITTED",
+        "description": "register request received",
+        "source": "automated",
+    },
+    "ERRORED": {
+        "previous_state": "SUBMITTED",
+        "current_state": "ERRORED",
+        "description": "registration request error",
+        "source": "automated",
+    },
+    "EMAIL_VERIFIED": {
+        "previous_state": "SUBMITTED",
+        "current_state": "EMAIL_VERIFIED",
+        "description": "completed email verification",
+        "source": "automated",
+    },
+    "PENDING_REVIEW": {
+        "previous_state": "EMAIL_VERIFIED",
+        "current_state": "PENDING_REVIEW",
+        "description": "review email sent",
+        "source": "automated",
+    },
+    "REVIEWED": {
+        "previous_state": "PENDING_REVIEW",
+        "current_state": "REVIEWED",
+        "description": "registration request received",
+        "source": "admin@rotational.io",
+    },
+    "ISSUING_CERTIFICATE": {
+        "previous_state": "REVIEWED",
+        "current_state": "ISSUING_CERTIFICATE",
+        "description": "issuing certificate",
+        "source": "automated",
+    },
+    "VERIFIED": {
+        "previous_state": "ISSUING_CERTIFICATE",
+        "current_state": "VERIFIED",
+        "description": "certificate issued",
+        "source": "automated",
+    },
+    "REJECTED": {
+        "previous_state": "PENDING_REVIEW",
+        "current_state": "REJECTED",
+        "description": "registration rejected",
+        "source": "admin@rotational.io",
+    },
+    "APPEALED": {
+        "previous_state": "REJECTED",
+        "current_state": "APPEALED",
+        "description": "registration appealed",
+        "source": "admin@rotational.io",
+    },
+}
+
+##########################################################################
+# Helper Methods - Used by subsequent functions to synthesize VASP records
+##########################################################################
+
+
+def fake_legal_name(vasp):
+    """
+    Given a string representing a VASP's name, return a valid
+    dictionary for the faked name identifiers
+    """
+    return {
+        "name_identifiers": [
+            {
+                "legal_person_name": vasp,
+                "legal_person_name_identifier_type": "LEGAL_PERSON_NAME_TYPE_CODE_SHRT",
```

Comment:
Could we change this to `LEGAL_PERSON_NAME_TYPE_CODE_LEGL` please? Sorry I didn't catch this before, the legal name is required, and it affects some of the name finding algorithms we use for the front end.